binaries/bzutil/main.go

```diff
@@ -130,6 +156,26 @@ func main() {
 		}
 		parseAndSetLevel(*actionLogLevel)
 		uploadBzAction(*actionAddr, *actionCommandDigest, *actionRootDigest, *actionNoCache, *actionJson)
+	} else if batchUpload.Parsed() {
+		parseAndSetLevel(*batchUploadLogLevel)
+		if strings.ToLower(strings.Trim(*batchUploadTest, " ")) == "true" && (*batchUploadDir != "") {
+			fmt.Printf("Supply either -dir or -test=true not both.\n")
+			return
+		}
+		digestsStr, e := batchUploadFiles(*batchUploadAddr, *batchUploadDir,
+			strings.ToLower(strings.Trim(*batchUploadTest, " ")) == "true")
+		if e != nil {
+			log.Fatalf("batch update returned %s", e.Error())
+		} else {
+			fmt.Printf("the following digests were uploaded:\n%s\n", digestsStr)
```

Comment:
This program should keep stdout machine readable - please just output newline-separated digests in the same format as other commands: "<hash>/<size>"

Reply:
Realizing now github reformatted my comment, the digests look like: "1b00e10d51c107c1a1f06ebdc09dea3c45e06fd257481d085d4e37566f6a6041/76"

Reply:
done